client/samples/lib/sample/vcenter/vm/create/create_default_vm.rb

```diff
@@ -0,0 +1,84 @@
+# Copyright 2014-2017 VMware, Inc.  All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+require 'sample/framework'
+require 'vapi'
+require 'com/vmware/vcenter/vapi'
+require 'sample/vcenter/helper/datastore_helper'
+require 'sample/vcenter/helper/cluster_helper'
+require 'sample/vcenter/helper/folder_helper'
+
+class CreateDefaultVM < SampleBase
+  
+  VCENTER_MODULE = Com::Vmware::Vcenter
+  VCENTER_VM_CLASS = VCENTER_MODULE::VM
+  VM_SPEC = Com::Vmware::Vcenter::VM::CreateSpec
+  VM_PLACEMENT_SPEC = Com::Vmware::Vcenter::VM::PlacementSpec
+  GUESTOS = Com::Vmware::Vcenter::Vm::GuestOS
```

Comment:
You can use the `VCENTER_MODULE` constant you made above on these constants as well.